pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -22,6 +22,7 @@
   libGL,
   buildExamples ? false,
   buildTests ? false,
+  debug ? false,
   developerBuild ? false,
```

Comment:
What is the difference between debug and developerBuild?

Reply:
According to the upstream build documentation, a "developer build" is an in-source build of Qt for developers. I have no idea why that would be useful to have in Nixpkgs, but I will make sure that we respect that flag for the purpose of propagating the correct CFLAGS.